### PR TITLE
Creation of image gallery 2

### PR DESCRIPTION
**Pull request by Avassaa**

### CODE REVIEW

File: editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx
Comment (Ayazelar) [inline editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx:1]: The image values must be come from spica storage

File: editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx [hunk 1]
```diff
@@ -0,0 +1,421 @@
...
+                            <div className={this.decorateCSS("gallery-item")} key={image.key} onClick={() => this.openModal(imageIndex)}>
+                                <div className={this.decorateCSS("image-container")}>
+                                    <img
+                                        src={image.value}
+                                        alt={image.displayer}
+                                        className={this.decorateCSS("gallery-image")}
+                                    />
+                                    <div className={this.decorateCSS("overlay")} />
+                                    <div className={this.decorateCSS("zoom-icon")}>
+                                        <ComposerIcon propsIcon={{ className: this.decorateCSS("magnifier-icon") }} name="FaSearch" />
```
Comment (Ayazelar) [inline editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx:393]: the name of icon must be come from prop

File: editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx [hunk 2]
```diff
@@ -0,0 +1,421 @@
...
+                    <div className={this.decorateCSS("modal")}>
+                        <div className={this.decorateCSS("modal-content")}>
+                            <span className={this.decorateCSS("close")} onClick={this.closeModal}>
+                                &times;
+                            </span>
+                            <span className={this.decorateCSS("prev")} onClick={this.prevImage}>
+                                &#10094;
+                            </span>
+                            <span className={this.decorateCSS("next")} onClick={this.nextImage}>
+                                &#10095;
```
Comment (Ayazelar) [inline editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx:410]: you should use composer icon component instead of &#10094; &#10095;

**Review by Ayazelar — CHANGES_REQUESTED** [review]
The changes to the image gallery and the feature12 should be separated into two separate processes.

File: editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx
Comment (Avassaa) [reply editor-components/Image_gallery/image_gallery_2/image-gallery2.tsx:1]: Do i have to initialize a spica storage myself or is there any source for that

**Comment by dogusmiuw** [comment]
Please exclude feature12 files and send this again

**Pull request closed without merging**